app/src/Model/Table/Article.php

```diff
@@ -3,19 +3,43 @@
 
 namespace App\Model\Entity;
 
+use Cake\Collection\Collection;
 use Cake\ORM\Entity;
 
+/**
+ * @property int $user_id
+ * @property string $title
+ * @property string $slug
+ * @property string $tag_string
+ * @property array $tags
```

Comment:
What kind of class will be set for the $tags? 

Please try to run the bake command again for model Articles and compare the generated from bake and current LOC.

Reply:
I just updated it to `\App\Model\Entity\Tag[]`